src/search/csearch.c

```
@@ -71,6 +71,19 @@ static uint64_t sr_parse_seed(const char *value, const char *invalid_msg)
   return (uint64_t)parsed;
 }
 
+static real sr_parse_positive_real(const char *value, const char *invalid_msg)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'sr_parse_positive_real' of similar type ('const char *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)